openlogin/account-page/index.test.ts

```diff
@@ -0,0 +1,342 @@
+import { expect, Page } from "@playwright/test";
+import { test } from "./index.lib";
+import { useAutoCancel2FASetup } from "../../utils";
+import { useAutoCancelShareTransfer } from "../../utils/index";
+import Mailosaur from "mailosaur";
+import { Link } from "mailosaur/lib/models";
+import { generate } from "generate-password";
+import { validateMnemonic } from "bip39";
+
+const mailosaur = new Mailosaur(process.env.MAILOSAUR_API_KEY || "");
+
+const testEmail = `accnt+hello+${Date.now()}@${
+  process.env.MAILOSAUR_SERVER_DOMAIN
+}`;
+
+const backupEmail = `hello+backup+${Date.now()}@${
+  process.env.MAILOSAUR_SERVER_DOMAIN
+}`;
+
+const randomPassword = generate({
+  length: 15,
+  numbers: true,
+  uppercase: true,
+  lowercase: true,
+  strict: true,
+});
+const newRandomPassword = generate({
+  length: 13,
+  numbers: true,
+  uppercase: true,
+  lowercase: true,
+  strict: true,
+});
+
+function findLink(links: Link[], text: string) {
+  for (const link of links) {
+    if (link.text === text) return link;
+  }
+  return null;
+}
+test.describe.serial("Account page test", () => {
+  let page: Page;
+  test.beforeAll(async ({ browser, openloginURL, user }) => {
+    const context = await browser.newContext();
```

Comment:
please extract out this process of passwordless login into a function in utils. Check the pattern which other logins follow. @sidharthWeb3auth

Reply:
Also I would suggest updating the "login with passwordless" test as well, to have a same flow everywhere.

Reply:
done